framework/mesh/mesh_continuum/grid_face_histogram.cc

```diff
@@ -17,7 +17,7 @@ GridFaceHistogram::NumberOfFaceHistogramBins() const
 size_t
 GridFaceHistogram::MapFaceHistogramBins(size_t num_face_verts) const
 {
-  size_t category_counter = -1;
+  int64_t category_counter = -1;
```

Comment:
how did you know that this integer should be a long one. it was `size_t `before. just curious

Reply:
`size_t` is unsigned, `int64_t` is signed. That was the problem. Assigning negative numbers into unsigned types is frowned upon by sanitizers....